src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -229,21 +242,23 @@ private static Publisher modifyPublisher(Publisher publisher, String id) {
     public Result<List<BidderBid>> makeBids(BidderCall<BidRequest> httpCall, BidRequest bidRequest) {
```

Comment:
Also, when you override `makeBidderResponse`, `makeBids` is no longer used. So, just remove this code entirely.

Reply:
Ok. Will do. Thx for advice.